### PR TITLE
add description for enable-table-lock

### PR DESCRIPTION
**Pull request by tangenta**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-configuration-file.md [hunk 1]
```diff
@@ -165,6 +165,13 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 + 如果确认环境存在网络隔离的可能，开启这个参数可以减少服务不可用的窗口期。
 + 如果无法准确判断隔离、网络中断、宕机等情况，这个机制存在误判情况从而导致可用性、性能降低。如果网络中从未发生过网络故障，不推荐开启此选项。
 
+### `enable-table-lock` <span class="version-mark">从 v4.0.0 版本开始引入</span>
+
```
Comment (lilin90) [inline tidb-configuration-file.md:169]:
```suggestion

> **警告：**
>
> LOCK TABLE 为实验特性，不建议在生产环境中使用。

```

File: tidb-configuration-file.md [hunk 2]
```diff
@@ -165,6 +165,13 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
++ 控制是否开启 LOCK TABLE 特性。
++ 默认值：false
++ TABLE LOCK 用于协调多个 session 之间对同一张表的并发访问。目前已支持的锁种类包括 `READ`、`WRITE` 和 `WRITE LOCAL`。当该配置项为 `false` 时，执行 `LOCK TABLE` 和 `UNLOCK TABLE` 语句不会生效，并且会报 "LOCK/UNLOCK TABLES is not supported" 的警告。
++ 注意 LOCK TABLE 为实验特性，不建议在生产环境中使用。
```
Comment (lilin90) [inline tidb-configuration-file.md:173]:
```suggestion
```

**Review by lilin90 — APPROVED** [review]
LGTM

File: tidb-configuration-file.md [hunk 3]
```diff
@@ -165,6 +165,16 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
 
+### `enable-table-lock` <span class="version-mark">从 v4.0.0 版本开始引入</span>
+
+> **警告：**
+>
+> LOCK TABLE 为实验特性，不建议在生产环境中使用。
+
++ 控制是否开启 LOCK TABLE 特性。
++ 默认值：false
++ TABLE LOCK 用于协调多个 session 之间对同一张表的并发访问。目前已支持的锁种类包括 `READ`、`WRITE` 和 `WRITE LOCAL`。当该配置项为 `false` 时，执行 `LOCK TABLE` 和 `UNLOCK TABLE` 语句不会生效，并且会报 "LOCK/UNLOCK TABLES is not supported" 的警告。
```
Comment (Oreoxmt) [inline tidb-configuration-file.md:176]:
```suggestion
+ LOCK TABLE 用于协调多个 session 之间对同一张表的并发访问。目前已支持的锁种类包括 `READ`、`WRITE` 和 `WRITE LOCAL`。当该配置项为 `false` 时，执行 `LOCK TABLE` 和 `UNLOCK TABLE` 语句不会生效，并且会报 "LOCK/UNLOCK TABLES is not supported" 的警告。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: experimental-features.md [hunk 4]
```diff
@@ -35,6 +35,7 @@ aliases: ['/docs-cn/dev/experimental-features-4.0/','/zh/tidb/dev/experimental-f
 + [生成列](/generated-columns.md#生成列)。（v2.1 实验特性）
 + [自定义变量](/user-defined-variables.md#用户自定义变量)。（v2.1 实验特性）
 + [Cascades Planner](/system-variables.md#tidb_enable_cascades_planner)：基于 Cascades 框架的自顶向下查询优化器。（v3.0 实验特性）
++ [LOCK TABLE](/tidb-configuration-file.md#enable-table-lock-从-v400-版本开始引入)（v4.0 实验特性）
 + [元数据锁](/metadata-lock.md) （v6.3.0 实验特性）
```
Comment (Oreoxmt) [inline experimental-features.md:39]:
```suggestion
+ [元数据锁](/metadata-lock.md)（v6.3.0 实验特性）
```

File: experimental-features.md [hunk 5]
```diff
@@ -35,7 +35,8 @@ aliases: ['/docs-cn/dev/experimental-features-4.0/','/zh/tidb/dev/experimental-f
 + [生成列](/generated-columns.md#生成列)。（v2.1 实验特性）
 + [自定义变量](/user-defined-variables.md#用户自定义变量)。（v2.1 实验特性）
 + [Cascades Planner](/system-variables.md#tidb_enable_cascades_planner)：基于 Cascades 框架的自顶向下查询优化器。（v3.0 实验特性）
-+ [元数据锁](/metadata-lock.md) （v6.3.0 实验特性）
++ [LOCK TABLE](/tidb-configuration-file.md#enable-table-lock-从-v400-版本开始引入)（v4.0 实验特性）
```
Comment (Oreoxmt) [inline experimental-features.md:38]:
```suggestion
+ [LOCK TABLE](/tidb-configuration-file.md#enable-table-lock-从-v400-版本开始引入)（v4.0.0 实验特性）
```

File: experimental-features.md [hunk 6]
```diff
@@ -35,7 +35,8 @@ aliases: ['/docs-cn/dev/experimental-features-4.0/','/zh/tidb/dev/experimental-f
 + [生成列](/generated-columns.md#生成列)。（v2.1 实验特性）
 + [自定义变量](/user-defined-variables.md#用户自定义变量)。（v2.1 实验特性）
 + [Cascades Planner](/system-variables.md#tidb_enable_cascades_planner)：基于 Cascades 框架的自顶向下查询优化器。（v3.0 实验特性）
-+ [元数据锁](/metadata-lock.md) （v6.3.0 实验特性）
++ [LOCK TABLE](/tidb-configuration-file.md#enable-table-lock-从-v400-版本开始引入)（v4.0.0 实验特性）
```
Comment (Oreoxmt) [inline experimental-features.md:38]:
```suggestion
+ [表级锁 (Table Locks)](/tidb-configuration-file.md#enable-table-lock-从-v400-版本开始引入)（v4.0.0 实验特性）
```

File: tidb-configuration-file.md [hunk 7]
```diff
@@ -165,6 +165,16 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 + 如果确认环境存在网络隔离的可能，开启这个参数可以减少服务不可用的窗口期。
 + 如果无法准确判断隔离、网络中断、宕机等情况，这个机制存在误判情况从而导致可用性、性能降低。如果网络中从未发生过网络故障，不推荐开启此选项。
 
+### `enable-table-lock` <span class="version-mark">从 v4.0.0 版本开始引入</span>
+
+> **警告：**
+>
+> LOCK TABLE 为实验特性，不建议在生产环境中使用。
```
Comment (Oreoxmt) [inline tidb-configuration-file.md:172]:
```suggestion
> 表级锁 (Lock Tables) 为实验特性，不建议在生产环境中使用。
```

File: tidb-configuration-file.md [hunk 8]
```diff
@@ -165,6 +165,16 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
+
++ 控制是否开启 LOCK TABLE 特性。
```
Comment (Oreoxmt) [inline tidb-configuration-file.md:174]:
```suggestion
+ 控制是否开启表级锁特性。
```

File: tidb-configuration-file.md [hunk 9]
```diff
@@ -165,6 +165,16 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
++ 默认值：false
++ LOCK TABLE 用于协调多个 session 之间对同一张表的并发访问。目前已支持的锁种类包括 `READ`、`WRITE` 和 `WRITE LOCAL`。当该配置项为 `false` 时，执行 `LOCK TABLE` 和 `UNLOCK TABLE` 语句不会生效，并且会报 "LOCK/UNLOCK TABLES is not supported" 的警告。
```
Comment (Oreoxmt) [inline tidb-configuration-file.md:176]:
```suggestion
+ 表级锁用于协调多个 session 之间对同一张表的并发访问。目前已支持的锁种类包括 `READ`、`WRITE` 和 `WRITE LOCAL`。当该配置项为 `false` 时，执行 `LOCK TABLE` 和 `UNLOCK TABLE` 语句不会生效，并且会报 "LOCK/UNLOCK TABLES is not supported" 的警告。
```

**Review by crazycs520 — APPROVED** [review]

**Review by lilin90 — APPROVED** [review]
LGTM

**Pull request merged**